src/lando/utils/github.py

```diff
@@ -554,11 +561,22 @@ def reviews(self) -> list:
 
     @property
     def commit_message(self) -> str:
-        """Return a string combining the pull request title, description, and URL."""
-        lines = [self.title, ""]
+        """Return a string combining the pull request title with reviewers, description, and URL."""
+
+        reviewers = [
+            GITHUB_REVIEWERS_MAP.get(u, u)
+            for u in self.reviews_summary
+            if self.reviews_summary.get(u) == self.Review.APPROVED
+        ]
+        approvals = []
+
+        lines = [replace_reviewers(self.title, reviewers, approvals), ""]
+
         if self.body:
             lines += [self.body, ""]
-        lines.append(f"Pull request: {self.html_url}")
+
+        lines.append(f"closes: {self.html_url}")
```

Comment:
Not sure if this change belongs in this PR? I am also wondering if it's sufficiently self-descriptive for someone looking at the commit history. Does it still work if it says `closes: pull request <url>`? Did we document somewhere what difference this would have vs. the current behaviour of triggering PR closing via the API?